lib/json/logger.ex

```diff
@@ -0,0 +1,48 @@
+defmodule Json.Logger do
```

Comment:
One last small detail, I know it might be against the convention but since the project has historically used the `JSON` namespace I would kindly ask you to rename this to `JSON.Logger` 🙏

Reply:
no, done :)